### DISCUSSION TITLE
How to make a 2-Column Form Responsive for Mobile?

### DISCUSSION THREAD

**Discussion by pbarmak** [Budibase/budibase#9852] [Help]
I have a screen with a main container and 2 sub-containers in order to create a 2-column form. It looks fine on desktop, but I don't know how to get it to go from 2 columns to a single, 100% width column (where the second child container drops below the first one). Do I need to have some custom CSS for mobile? Or am I even setting this up right by using child containers? I tried with a section, but that didn't work either.

Here is the form:
![image](https://user-images.githubusercontent.com/45037626/222276607-05f17af2-a3a4-47cc-9a17-2a8797c26c56.png)

And this is what it looks like on the mobile view (same thing if I preview the app and use Firefox's responsive design mode):
![image](https://user-images.githubusercontent.com/45037626/222276834-6c3bc884-6373-4817-b280-efb2b6e09b3d.png)

It doesn't shift the containers so that the Left Container takes up 100% of the width and the Right Container then drops down below it (also with 100% width).

Thanks for any pointers. I'd love to have multi-column forms that also work well in mobile.

**Comment by melohagan** [comment]
Hey @pbarmak 

Have you looked at adding a two column section? https://docs.budibase.com/docs/using-sections

**Reply by pbarmak** [reply]
Hey @melohagan, yes, I originally had it using a section. But even that would not switch to full width on mobile. Works great on desktop, but mobile doesn't re-style to full width.

**Comment by melohagan** [comment]
Hey @pbarmak 

I've tested this on Firefox for version **2.4.8** and I'm not seeing any issues:
![Screenshot 2023-03-09 at 14 33 30](https://user-images.githubusercontent.com/101575380/224056661-797fa12b-416a-4099-8575-6a4c582de1f7.png)

**Mobile**
![Screenshot 2023-03-09 at 14 33 48](https://user-images.githubusercontent.com/101575380/224056738-70296e42-8080-4cad-8bf4-32e0a1577072.png)

Are you sure your *Extra Styles* embed isn't affecting things?
